cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala

```diff
@@ -118,50 +172,16 @@ object CassandraLauncher {
       require(cassandraDirectory.mkdirs(), s"Couldn't create Cassandra directory [$cassandraDirectory]")
   }
 
-  private def startEmbedded(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int): Unit = {
-    prepareCassandraDirectory(cassandraDirectory, clean)
-
-    val realPort = if (port == 0) randomPort else port
-    val storagePort = freePort()
-
-    // http://wiki.apache.org/cassandra/StorageConfiguration
-    val conf = readResource(configResource)
-    val amendedConf = conf
-      .replace("$PORT", realPort.toString)
-      .replace("$STORAGE_PORT", storagePort.toString)
-      .replace("$DIR", cassandraDirectory.getAbsolutePath)
-    val configFile = new File(cassandraDirectory, configResource)
-    writeToFile(configFile, amendedConf)
-
-    System.setProperty("cassandra.config", "file:" + configFile.getAbsolutePath)
-    System.setProperty("cassandra-foreground", "true")
-
-    // runManaged = true to avoid System.exit
-    val daemon = new CassandraDaemon(true)
-    daemon.activate()
-    cassandraDaemon = Some(new Closeable {
-      override def close(): Unit = daemon.deactivate()
-    })
-  }
-
-  private def startForked(cassandraDirectory: File, configResource: String, realPort: Int): Unit = {
+  private def startForked(cassandraDirectory: File, configFile: File, cassandraBundle: File, classpath: immutable.Seq[String], port: Int): Unit = {
```

Comment:
I don't think so - embedding doesn't work on Windows (because Cassandra always calls System.exit when Cassandra shuts down on Windows), and it can't be restarted after it's been started.